apps/challenges/views.py

```diff
@@ -2740,3 +2883,15 @@ def manage_worker(request, challenge_pk, action):
             response_data = {"action": "Failure", "error": message}
 
     return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["GET"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_all_challenge_templates(request):
+    q_params = {"is_active": True}
```

Comment:
do we need any checks for users here? or will this api be accessible for everyone
cc: @RishabhJain2018